package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "electric-moray",
     "description": "Manta directory API Moray proxy service",
-    "version": "1.1.0",
+    "version": "1.1.1",
```

Comment:
Nit: could justify a 1.2.0 (minor version bump) for this, given support for fast v1 and v2 inbound.

Also a CHANGES.md potentially here to explain the server fast protocol change.